src/components/ReportActionsSkeletonView/index.tsx

```diff
@@ -1,25 +1,16 @@
-import PropTypes from 'prop-types';
 import React from 'react';
 import {Dimensions, View} from 'react-native';
 import CONST from '@src/CONST';
 import SkeletonViewLines from './SkeletonViewLines';
 
-const propTypes = {
-    /** Whether to animate the skeleton view */
-    shouldAnimate: PropTypes.bool,
-
-    /** Number of possible visible content items */
-    possibleVisibleContentItems: PropTypes.number,
-};
-
-const defaultProps = {
-    shouldAnimate: true,
-    possibleVisibleContentItems: 0,
+type ReportActionsSkeletonViewProps = {
+    shouldAnimate?: boolean;
```

Comment:
@JKobrynski  maybe add comments that were mentioned in propTypes for these values

Reply:
Done!